packages/mui-base/vitest.config.ts

```diff
@@ -1,4 +1,11 @@
 import { mergeConfig, defineProject } from 'vitest/config';
 import sharedConfig from '../../vitest.shared';
 
-export default mergeConfig(sharedConfig, defineProject({}));
+export default mergeConfig(
+  sharedConfig,
+  defineProject({
+    define: {
+      'process.env.NODE_ENV': JSON.stringify('test'),
```

Comment:
@JCQuintas, you may need to do something similar to avoid errors like https://app.circleci.com/pipelines/github/mui/base-ui/4624/workflows/401822a3-1f69-4ee6-a26f-c8e9d6bc45c5/jobs/31876